sphinxcontrib/mermaid.py

```diff
@@ -166,7 +167,11 @@ def render_mm(self, code, options, _fmt, prefix='mermaid'):
     with open(tmpfn, 'w') as t:
         t.write(code)
 
-    mm_args = [mermaid_cmd, '-i', tmpfn, '-o', outfn]
+    if isinstance(mermaid_cmd, str):
+        mm_args = shlex.split(mermaid_cmd)
+    else:
+        mm_args = list(mermaid_cmd)
+    mm_args += ['-i', tmpfn, '-o', outfn]
     mm_args.extend(self.builder.config.mermaid_params)
```

Comment:
@slanzmich isn't the same than directly add the extra params in the `mermaid_params` config? 

`mermaid_params = [...] `

Reply:
Users cannot add the `-i` / `-o` commands, because they depend on the current input file and the temporary directory.

It is however not a problem if users specify these parameters in their command, because they will be overwritten by the ones we append. I.e.
```bash
npx mmdc -i input_1.md -o output_1.png -i input_2.md -o output_2.png
```
will convert `input_2.md` to `output_2.png`.

Reply:
Sorry, I misread your comment and confused `mermaid_params` with `mermaid_cmd` option.

Adding any additional parameters to `mermaid_params` does not work for my use case, because the `-i` / `-o` options are inserted first, such that they are passed to the wrong command (`npx` instead of `mmdc` in my case).

In fact, the `-i` / `-o` options should be added after the `mermaid_params` to prevent users from accidentally messing with these parameters. If that is changed, my change would be optional in the sense that I could also configure
```python
mermaid_cmd = "npx"
mermaid_params = ["--no-install", "mmdc"]
```
However, the fact that this works is an implementation detail and feels wrong. The `mermaid_cmd` is actually `npx --no-install mmdc`, the arguments are no arguments to `mermaid`.

Reply:
I pushed a few more commits. Let me know what you think and if I should squash any of them.